map/src/dialogs/favorites/DeleteFavoriteDialog.jsx

```diff
@@ -93,13 +93,15 @@ export default function DeleteFavoriteDialog({ dialogOpen, setDialogOpen, wpt =
     }
 
     return (
-        <Dialog open={true} onClose={toggleShowDialog} onClick={(e) => e.stopPropagation()}>
+        <Dialog id="se-delete-fav-dialog" open={true} onClose={toggleShowDialog} onClick={(e) => e.stopPropagation()}>
             <DialogTitle>{getTitleDialog()}</DialogTitle>
             <DialogContent>
                 <DialogContentText>{getQuestionDialog()}</DialogContentText>
             </DialogContent>
             <DialogActions>
-                <Button onClick={() => deleteWpt()}>Delete</Button>
+                <Button id="se-delete-fav-dialog-submit" onClick={() => deleteWpt()}>
```

Comment:
When you don't need any params for event handler, please contract the call:

`onClick={() => deleteWpt()}` -> onClick={deleteWpt}

The same as you've already done at line 96 for onClose.